services/github/github-latest-branch-release.tester.js

```diff
@@ -0,0 +1,33 @@
+import Joi from 'joi'
+import { isSemver } from '../test-validators.js'
+import { ServiceTester } from '../tester.js'
+
+export const t = new ServiceTester({
+  id: 'GithubLatestBranchRelease',
+  title: 'Github Latest Branch Release',
+  pathPrefix: '/github',
+})
+
+t.create('Release')
+  .get('/v/release/expressjs/express.json')
```

Comment:
If we end up coming back to this, none of the tests in this file are actually testing the service class that has been added in this PR, but lets not get bogged down in that for the moment